src/vanilla.ts

```diff
@@ -146,6 +146,7 @@ const buildProxyFunction = (
       }
       Object.defineProperty(snap, key, desc)
     })
+    Object.preventExtensions(snap)
     return snap
```

Comment:
Just a little bit compress.
```suggestion
    return Object.preventExtensions(snap)
```